src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/RandomTestData.kt

```diff
@@ -34,6 +34,12 @@ fun randomCro(): String {
 fun randomName(): String = randomLowerCaseString()
 fun randomNationalInsuranceNumber(): String = UUID.randomUUID().toString()
 fun randomDriverLicenseNumber(): String = UUID.randomUUID().toString()
+fun randomBirthPlace(): String = randomLowerCaseString()
+fun randomBirthCountry(): String = randomLowerCaseString()
+fun randomNationality(): String = randomLowerCaseString()
+fun randomReligon(): String = randomLowerCaseString()
+fun randomSexualOrientation(): String = randomLowerCaseString()
+fun randomSex(): String = randomLowerCaseString()
```

Comment:
are these all free text strings, or are some of them codes? eg is birthCountry ISO country code? It will still be a string to us of course

Reply:
have removed birth country / birth place as it was not in the response for prisoner search